internal/config/mapper_test.go

```diff
@@ -219,3 +219,93 @@ func TestToCommandProto(t *testing.T) {
 		})
 	}
 }
+
+func TestMapper_ToAgentConfigLogProto(t *testing.T) {
+	tests := []struct {
+		log      *Log
+		expected *mpi.Log
+		name     string
+	}{
+		{
+			name: "Test 1: Log level DEBUG",
+			log: &Log{
+				Level: "DEBUG",
+			},
+			expected: &mpi.Log{
+				LogLevel: mpi.Log_LOG_LEVEL_DEBUG,
+				LogPath:  "",
+			},
+		},
+		{
+			name: "Test 2: Log level INFO",
+			log: &Log{
+				Level: "INFO",
+			},
+			expected: &mpi.Log{
+				LogLevel: mpi.Log_LOG_LEVEL_INFO,
+				LogPath:  "",
+			},
+		},
+		{
+			name: "Test 3: Log level WARN",
+			log: &Log{
+				Level: "WARN",
+			},
+			expected: &mpi.Log{
+				LogLevel: mpi.Log_LOG_LEVEL_WARN,
+				LogPath:  "",
+			},
+		},
+		{
+			name: "Test 4: Log level ERROR",
+			log: &Log{
+				Level: "ERROR",
+			},
+			expected: &mpi.Log{
+				LogLevel: mpi.Log_LOG_LEVEL_ERROR,
+				LogPath:  "",
+			},
```

Comment:
Can there be tests where log level is greater than 3 and what if there's blank level?